scss/_patterns_hero.scss

```diff
@@ -0,0 +1,28 @@
+@mixin vf-p-hero {
+  .p-hero {
+    @extend %vf-row;
+    @extend %section-padding--default;
+
+    // same as padding on %section-padding--default
+    row-gap: $spv--strip-regular * 0.5;
+
+    @media (min-width: $breakpoint-large) {
+      row-gap: $spv--strip-regular;
+    }
+
+    .p-hero__header,
+    .p-hero__footer,
+    .p-hero__content {
+      grid-column-end: 13;
+      grid-column-start: 4;
+    }
```

Comment:
If these are all doing the same thing then maybe a single wrapper class?

Reply:
Also I think on small screens we should not contain the content like this.

Reply:
> If these are all doing the same thing then maybe a single wrapper class?

Exactly, I am considering if there are any differences that justify the "fake" semantics there.
The only point so far is that you should not have more that these three in single hero. So with header/content/footer we better enforce this structure, but it may be simply documented as not allowing more than 3 `p-hero__content` blocks for example.

Reply:
Although we may take the wrapper type into account on different layouts (like when putting header on left):
https://vanilla-framework-4838.demos.haus/docs/examples/brochure/hero-25-75
https://vanilla-framework-4838.demos.haus/docs/examples/brochure/hero-50-50